netmaster/netwrapper.go

```diff
@@ -0,0 +1,72 @@
+/***
```

Comment:
@vvb Thanks for making this change.
Would it be possible to move this file to a common package like /utils/netutils so that we can reuse it in other places?